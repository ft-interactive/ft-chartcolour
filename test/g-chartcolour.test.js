import gChartcolour from '../';

it('should have colours', () => {
  expect(gChartcolour).toMatchSnapshot();
});
