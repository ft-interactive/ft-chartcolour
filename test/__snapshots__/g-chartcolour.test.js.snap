// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have colours 1`] = `
Object {
  "background": Object {
    "clean": "#FFF",
    "print": "#FEE",
    "social": "#333",
    "video": "#335",
    "web": "#FFF1e5",
  },
  "barPrint": Array [
    "#0083b3",
    "#95bfc5",
    "#004d6d",
    "#64a2c1",
    "#c6d6cc",
  ],
  "basicLinePrint": Array [
    "#c6d6cc",
    "#006a93",
  ],
  "basicLineSocial": Array [
    "#81838F",
    "#EB3F50",
  ],
  "basicLineWeb": Array [
    "#EB5E8D",
    "#0f5499",
  ],
  "categorical_bar": Array [
    "#1E558C",
    "#94d2e6",
    "#1E8FCC",
    "#B3325D",
    "#FF75A3",
    "#D9CCC3",
    "#AECC70",
    "#F34D5B",
  ],
  "categorical_line": Array [
    "#0f5499",
    "#EB5E8D",
    "#70DCE6",
    "#9dbf57",
    "#208fce",
    "#7f062e",
    "#c2b7af",
    "#ce3140",
  ],
  "diverging": Array [
    "#b31147",
    "#d44d41",
    "#e67f64",
    "#f1ae92",
    "#f3dec8",
    "#95c8d4",
    "#5aa8cf",
    "#2f85c3",
    "#1162b3",
  ],
  "diverging_3": Array [
    "#b31147",
    "#cec6b9",
    "#105ba6",
  ],
  "diverging_5": Array [
    "#b31147",
    "#d75d79",
    "#cec6b9",
    "#5884d6",
    "#105ba6",
  ],
  "diverging_7": Array [
    "#b31147",
    "#d75d79",
    "#f09ba8",
    "#f3dec8",
    "#a9b0dd",
    "#5884d6",
    "#105ba6",
  ],
  "extendedLine": Array [
    "#d5ccbf",
    "#ed588d",
    "#FF0000",
    "#00FF00",
  ],
  "germanPoliticalParties_bar": Object {
    "AfD": "#1E8FCC",
    "CDU/CSU": "#33302E",
    "FDP": "#fcc83c",
    "Green": "#AECC70",
    "Left": "#B3325D",
    "Other": "#CEC6B9",
    "SPD": "#F34D5B",
  },
  "germanPoliticalParties_bar_social": Object {
    "AfD": "#52c0ff",
    "CDU/CSU": "#000000",
    "FDP": "#f5bb00",
    "Green": "#00aa5b",
    "Left": "#73008a",
    "Other": "#CEC6B9",
    "SPD": "#b80000",
  },
  "germanPoliticalParties_line": Object {
    "AfD": "#52c0ff",
    "CDU/CSU": "#000000",
    "FDP": "#f5bb00",
    "Green": "#00aa5b",
    "Left": "#73008a",
    "Other": "#CEC6B9",
    "SPD": "#b80000",
  },
  "linePrint": Array [
    "#006a93",
    "#ddb831",
    "#aad1d1",
    "#ad1c21",
    "#55a2c7",
    "#0083b3",
  ],
  "lineSocial": Array [
    "#ff0055",
    "#0066ff",
    "#b1e645",
    "#00ccff",
    "#f2e5da",
    "#65798c",
  ],
  "lineWeb": Array [
    "#0f5499",
    "#EB5E8D",
    "#70DCE6",
    "#9dbf57",
    "#208fce",
    "#7f062e",
    "#c2b7af",
  ],
  "mutedFirstLinePrint": Array [
    "#999999",
    "#006a93",
    "#ddb831",
    "#aad1d1",
    "#ad1c21",
    "#55a2c7",
    "#0083b3",
  ],
  "mutedFirstLineSocial": Array [
    "#cccccc",
    "#eb3f50",
    "#00d9ca",
    "#bf9413",
    "#1f5e99",
    "#a7ff59",
    "#ff9b96",
    "#81838f",
  ],
  "mutedFirstLineWeb": Array [
    "#B3A9A0",
    "#0f5499",
    "#EB5E8D",
    "#70DCE6",
    "#9dbf57",
    "#208fce",
    "#7f062e",
  ],
  "noData": Object {
    "area": "#DAD2C3",
    "line": "#B9B3A7",
  },
  "sequentialMulti": Array [
    "#f3dec8",
    "#98c3ce",
    "#649ec1",
    "#3c78ae",
    "#0f5499",
  ],
  "sequentialMulti_2": Array [
    "#f3dec8",
    "#f1b499",
    "#e88a6e",
    "#d7604c",
    "#bf3536",
    "#990f3d",
  ],
  "sequentialMulti_3": Array [
    "#f3dec8",
    "#8ac090",
    "#4c9298",
    "#236386",
    "#0a3866",
  ],
  "sequentialSingle": Array [
    "#d6d3ea",
    "#adb8e6",
    "#849cdb",
    "#5e82c8",
    "#3968ad",
    "#1b4f8d",
    "#0a3866",
  ],
  "sequentialSingle_teal": Array [
    "#f3dec8",
    "#96c8c7",
    "#5aa8ae",
    "#2e8790",
    "#0d656f",
    "#00464d",
  ],
  "ukPoliticalParties": Object {
    "alliance": "#ffc660",
    "conservative": "#6da8e1",
    "dup": "#827996",
    "green": "#65a68c",
    "independent": "#CEC6B9",
    "labour": "#e25050",
    "liberal democrats": "#f0a330",
    "other": "#CEC6B9",
    "pc": "#99d2d0",
    "sdlp": "#bed676",
    "sf": "#99bf70",
    "snp": "#f2d92d",
    "ukip": "#ca6dbf",
    "uup": "#6da8e1",
  },
  "ukPoliticalPartiesPrint": Object {
    "alliance": "#ffc660",
    "conservative": "#6D98D3",
    "dup": "#ABA0D0",
    "green": "#447A69",
    "independent": "#D1D0D0",
    "labour": "#E35453",
    "liberal democrats": "#EC812A",
    "other": "#CEC6B9",
    "pc": "#6ABDB6",
    "sdlp": "#B2CD47",
    "sf": "#71A04A",
    "snp": "#FCCE26",
    "ukip": "#7265AE",
    "uup": "#80A9AC",
  },
  "ukPoliticalPartiesText": Object {
    "alliance": "#dba436",
    "conservative": "#1369bf",
    "dup": "#443e71",
    "green": "#008066",
    "independent": "#a7a59b",
    "labour": "#bf1f1f",
    "liberal democrats": "#c68024",
    "other": "#a7a59b",
    "pc": "#54a19c",
    "sdlp": "#9cb157",
    "sf": "#6a8d32",
    "snp": "#99891b",
    "ukip": "#b3009d",
    "uup": "#1369bf",
  },
  "usPoliticalPartiesLargeArea": Object {
    "democrat": "#579dd5",
    "green": "#55c72c",
    "leaning democrat": "#a2c1e1",
    "leaning republican": "#f4a098",
    "libertarian": "#6e4794",
    "republican": "#e03d46",
    "swing": "#fcc83c",
  },
  "usPoliticalPartiesMidterms2018": Object {
    "democrat": "#0F5499",
    "republican large area": "#FF4C55",
    "republican small area": "#E60008",
  },
  "usPoliticalPartiesSmallArea": Object {
    "democrat": "#238fce",
    "green": "#47aa1d",
    "leaning democrat": "#a2c1e1",
    "leaning republican": "#f4a098",
    "libertarian": "#6e4794",
    "republican": "#e5262d",
    "swing": "#fcc83c",
  },
}
`;
